java/src/jmri/jmrit/dispatcher/DispatcherBundle.properties

```diff
@@ -111,6 +111,24 @@ StartingBlockBoxHint = Select the Block where the Train is located when Transit
 DestinationBlockBoxLabel = Destination Location of Train
 DestinationBlockBoxHint = Select the Block where the Train will be located when Transit ends.
 AutoRun = Run Train Automatically
+AllocateMethodLabel = Allocation Method
+AllocateMethodBoxHint = Number of Blocks ahead to auto allocate
+3Ahead = 3 Ahead
+AsFarAsPos = As Far As Pos
+ToSafeSections = To Safe Secs
+1Ahead = 1 Ahead
+2Ahead = 2 Ahead
+4Ahead = 4 Ahead
+5Ahead = 5 Ahead
+6Ahead = 6 Ahead
+AllocateAllTheWay = Allocate Train as far as possible
+AllocateAllTheWayBoxHint = Check for train to be allocated from start to end as far as possible. Leave unchecked to allocate no more than three Sections ahead.
```

Comment:
This key is listed twice in this file (lines 125 and 134); one of these needs to be removed to avoid errors later.

Reply:
Weird. OK will do